### PR TITLE
RWC2019: Add Options message and LabeledImage

### PR DESCRIPTION
**Pull request by LoyVanBeek**
- LabeledImage allows to explicitly pair an image and a label. `master` currently optionally pairs these two via the Image's frame_id but that feels a bit like a hack. 
- Options allows to ask a question and then suggest a reply via a custom keyboard in Telegram.

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -2,4 +2,24 @@ cmake_minimum_required(VERSION 2.8.3)
...
+
+# Generate added messages and services with any dependencies listed here
+generate_messages(
+  DEPENDENCIES
+  sensor_msgs
+)
+
+catkin_package(
+  CATKIN_DEPENDS message_runtime sensor_msgs
+)
```
Comment (MatthijsBurgh) [inline CMakeLists.txt]: Fix EOF

File: msg/Options.msg [hunk 2]
```diff
@@ -0,0 +1,2 @@
+string question
+string[] options
```
Comment (MatthijsBurgh) [inline msg/Options.msg]: Fix EOF

**Review by MatthijsBurgh — CHANGES_REQUESTED** [review]
Please fix the EOFs, and separate msg package would be better

**Comment by LoyVanBeek** [comment]
Maybe the messages would alsually better fit in the HMI msgs?

**Comment by ar13pit** [comment]
> Maybe the messages would alsually better fit in the HMI msgs?

Only if other clients are using it. Else it wouldn't make sense to generalize it yet.

File: CMakeLists.txt (hunk 1 above)
Comment (JosjaG) [reply CMakeLists.txt]: fixed

File: msg/Options.msg (hunk 2 above)
Comment (JosjaG) [reply msg/Options.msg]: fixed

**Review by MatthijsBurgh — APPROVED** [review]

**Comment by MatthijsBurgh** [comment]
@JosjaG you can merge this if CI succeeds and you agree with my last commit

**Pull request merged**